src/organisms/feedPageScrollableContent/feedPageScrollableContent.tsx

```diff
@@ -2,27 +2,32 @@ import React from 'react';
 import { useTheme } from 'styled-components';
```

Comment:
@shan8851 why do we want to have this organism here ? actually i cant remember that we had a task for it , its really comfortable to have it here ? i mean its contains a lot of logic , and i try to test it here, its not really working , so i dont really understand what is this.

lets have a call during the Monday to catch up and understand , maybe i am missing something.

Reply:
@goshatravin I think I mentioned last week that I started creating the scrollable content inside component lib but it might not actually make sense for it to be there and perhaps should just be in the new repo for feed page layout.

The story is overly complex for sure, just trying to replicate how it might be in the feed page, which is probably too specific for storybook.